test/release/examples/users-guide/base/configs.execopts

```diff
@@ -0,0 +1,4 @@
+./a.out --n 1000
+./a.out --n=1000
+./a.out --epsilon=0.000001 --debug=true  # configs-eps+debug.good
+./a.out --epsilon=0.000001 --debug       # configs-eps+debug.good
```

Comment:
Yeah, but we don't control literalinclude -- that's part of Sphinx's functionality (and I don't think we want to modify Sphinx...  And I'd still need to have -fake files for the compopts files...

I think the "right" thing to do is to have the test system do some stripping of prefixes/suffixes and diff that the fake and real files match... but I didn't have the scripting skills to do that in the time I've had so far...